data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/importer/CsaProfileCracImporter.java

```diff
@@ -39,16 +39,34 @@ public String getFormat() {
     @Override
     public CsaProfileCrac importNativeCrac(InputStream inputStream) {
         TripleStore tripleStoreCsaProfile = TripleStoreFactory.create(CsaProfileConstants.TRIPLESTORE_RDF4J_NAME);
-        ZipInputStream zipInputStream = new ZipInputStream(inputStream);
         ZipEntry zipEntry;
-        try {
-            while ((zipEntry = zipInputStream.getNextEntry()) != null) {
+        try (ZipInputStream zipInputStream = new ZipInputStream(inputStream)) {
+            //max number of entries and max size of entry are checked to avoid ddos attack with malicious zip file
+            int maxNbEntries = 200;
+            int maxSizeEntry = 10000000;
```

Comment:
maybe we can think of something configurable on application side ? i am not sure 10 megabytes are enough for real use cases

Reply:
ok, I change it to 10 gigabytes and add a "to do" to parameterize it